product_docs/docs/pgd/5/upgrades/index.mdx

```diff
@@ -111,6 +111,28 @@ Upgrading CAMO-Enabled Cluster requires upgrading CAMO groups one by one, while
 disabling the CAMO protection for the group being upgraded and reconfiguring it
 using the new [Commit Scope](../durability/commit-scopes) based settings.
 
+The following approach is recommended for upgrade two BDR nodes that
+consitute a CAMO pair:
+
+- Ensure `bdr.enable_camo` remains `off` for transactions on any of
+  the two nodes or redirect clients away from the two nodes. Removing
+  the CAMO pairing while attempting to use CAMO will lead to errors
+  and prevent further transactions.
+- Uncouple the pair by deconfiguring CAMO either by resetting
+  `bdr.camo_origin_for` and `bdr.camo_parter_of` (when upgrading from
+  BDR 3.7.x) or by using `bdr.remove_camo_pair` (on BDR 4.x).
+- Upgrade the two nodes to BDR 5.0
+- If not already existent, create a dedicated node group for the two
```

Comment:
Is the condition necessary here? Would simply "Create a dedicated node group" suffice.

(Looking to keep directions simple and as straight lined as possible)

Reply:
I guess most customers would not have a node group already, so yes, good to remove that.